pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/assets/manifests/triliovault-ovh-s3-target.yaml

```diff
@@ -0,0 +1,15 @@
+kind: Target
```

Comment:
apiVersion: triliovault.trilio.io/v1
kind: Target

Reply:
Thank you for trying this out. Updated it now.